### PR TITLE
[Merged by Bors] - Fix failing TestBlockBuilder_CreateBlockFlow

### PR DESCRIPTION
**Pull request by nkryuchkov**
## Motivation

WIP on https://github.com/spacemeshos/pm/issues/40

## Changes

Fix failing `TestBlockBuilder_CreateBlockFlow`

## Test Plan

- [x] `TestBlockBuilder_CreateBlockFlow` passes

## TODO

- [x] Finish

## DevOps Notes

- [ ] Does this code require configuration changes?
- [ ] Does this code affect public APIs?
- [ ] Does this code assume a new version of external services (PoET, elasticsearch, etc.)
- [ ] Does this code make changes to log messages that our monitoring infrastructure may rely on?

### CODE REVIEW

**Comment by nkryuchkov** [comment]
bors try

**Review by antonlerner — APPROVED** [review]

File: miner/builder_test.go [hunk 1]
```diff
@@ -269,7 +271,7 @@ func TestBlockBuilder_CreateBlockFlow(t *testing.T) {
 		assert.True(t, ContainsAtx(b.ATXIDs, atxs[2].ID()))*/
 
 		assert.Equal(t, []types.ATXID{atx1, atx2, atx3, atx4, atx5}, *b.ActiveSet)
-	case <-time.After(1 * time.Second):
+	case <-time.After(1 * time.Minute):
```
Comment (noamnelke) [inline miner/builder_test.go:274]: The other fix makes a lot of sense to me. I'm also ok with increasing the timeout, but I'm wondering if it isn't just a leftover from a first attempt at a fix - does it really take more than a second sometimes?

**Review by noamnelke — APPROVED** [review]

File: miner/builder_test.go (hunk 1 above)
Comment (nkryuchkov) [reply miner/builder_test.go:274]: @noamnelke Thanks for noticing. No, it's not a leftover. Even with the other fix, sometimes, under high CPU load, spawning a goroutine takes multiple seconds, so the timeout occurs. I've run the test around 5M times and it took around 15 seconds around 2 times per 1M iterations. So, I think, 1 minute would give some reserve.

**Review by antonlerner — APPROVED** [review]

**Comment by nkryuchkov** [comment]
bors merge

**Pull request closed without merging**